config/checkstyle-input-suppressions.xml

```diff
@@ -68,52 +68,7 @@
           files="[\\/]packageobjectfactory[\\/]zoo[\\/]FooCheck.java"/>
 
   <!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->
-  <suppress id="UnnecessaryOkComment"
```

Comment:
Everything lgtm....

Reply:
Thanks for reminding me. Later, I will comment on issue ticket @MANISH-K-07